tests/integration/test_sharepoint.py

```diff
@@ -57,6 +63,11 @@ def test_sharepoint_to_df_task():
     os.remove("Questionnaires.xlsx")
 
 
+def test_download_file(sharepoint):
```

Comment:
Please add docstring with raises section

Reply:
We are checking here if the exception is raised and not raising a new exception.